### PR TITLE
added contributors.md

### PR DESCRIPTION
**Pull request by shraddhavp**
#107 
  - Added contributors.md file 

Fixes: #107 

# Added the contributors.md file
 
 
# Additional Info
  - Anything related Issues

### CODE REVIEW

File: mysite/contributors.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# **OUR TOP CONTRIBUTORS !!!!**
+![Thank you Image](https://pvsmt99345.i.lithium.com/t5/image/serverpage/image-id/11515i5FE675D18BD5D6D4/image-size/large?v=1.0&px=999)
+
+# Top contributers!!:+1
```
Comment (Kajol-Kumari) [inline mysite/contributors.md:4]: Please provide a space and correct the typo here i.e `Top contributors!! :+1: `

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Minor changes @shraddhavp Rest LGTM :)

**Comment by shraddhavp** [comment]
made the made the changes  requested @Kajol-Kumari

File: mysite/contributors.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# **OUR TOP CONTRIBUTORS !!!!**
+![Thank you Image](https://pvsmt99345.i.lithium.com/t5/image/serverpage/image-id/11515i5FE675D18BD5D6D4/image-size/large?v=1.0&px=999)
+
+# Top contributors!!  :+1
```
Comment (Kajol-Kumari) [inline mysite/contributors.md:4]: Hey @shraddhavp I am still unable to see the emoji here, probably you missed the `:` here i.e ` :+1: ` Please add the colon after `1`.

**Review by Kajol-Kumari — COMMENTED** [review]
@shraddhavp Please move this file in the main directory i.e in parallel to `readme.md`

**Pull request closed without merging**